crates/anvil/tests/it/sign.rs

```diff
@@ -88,7 +92,7 @@ async fn can_sign_typed_data() {
         .unwrap();
     assert_eq!(
       signature,
-      "0x6ea8bb309a3401225701f3565e32519f94a0ea91a5910ce9229fe488e773584c0390416a2190d9560219dab757ecca2029e63fa9d1c2aebf676cc25b9f03126a1b".to_string()
+      "0x6ea8bb309a3401225701f3565e32519f94a0ea91a5910ce9229fe488e773584c0390416a2190d9560219dab757ecca2029e63fa9d1c2aebf676cc25b9f03126a01".to_string()
```

Comment:
is this the last blocker?

Reply:
Yes. The reason is that the last byte is not having + 27 appended. This is being fixed through https://github.com/alloy-rs/core/pull/499 and a followup on `alloy-rs/alloy` after.

Reply:
fixed! https://github.com/foundry-rs/foundry/pull/6808/commits/07fb76d550b06a8235423b77970c69365ca02ac4